recipes/ansible/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "ansible" %}
+{% set version = "2.3.1.0" %}
+{% set sha256 = "cd4b8f53720fcd0c351156b840fdd15ecfbec22c951b5406ec503de49d40b9f5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - paramiko
+    - jinja2
+    - pyyaml
+    - httplib2
+    - pycrypto
+    - six
```

Comment:
Usually, when using `--single-version-externally-managed`, we can remove the `run` time dependencies from the `build` section. Can you try removing the following:

```yaml
    - paramiko
    - jinja2
    - pyyaml
    - httplib2
    - pycrypto
    - six
```

and see what happens? (Sometimes the package imports itself in `setup.py` and the `run` time deps will be needed though.)